ui/admin/app/components/worker-diagram/index.hbs

```diff
@@ -1,3 +1,10 @@
 <div data-test-worker-diagram>
-  <WorkerDiagram::SingleFilter @egressFilter={{@egressFilter}} />
+  {{#if (feature-flag 'target-worker-filters-v2-ingress')}}
+    <WorkerDiagram::DualFilter
+      @egressFilter={{@egressFilter}}
+      @ingressFilter={{@ingressFilter}}
+    />
+  {{else}}
+    <WorkerDiagram::SingleFilter @egressFilter={{@egressFilter}} />
+  {{/if}}
```

Comment:
My initial plan was to have 3 logical paths but I went a different path for `DualFilter::HCP` so we only have 2 paths here. This now makes this logic overkill. We could easily move this fairly simple logic to the `target/details/index.hbs` template. I would like to hear other's thoughts on this.

Reply:
@cameronperera. So what would be the outcome of the logic inside of the `target/detail` are you calling `<DualFilter />` and  making decisions about `ent` and `hcp`  in there?

Reply:
In this [PR](https://github.com/hashicorp/boundary-ui/pull/1585), you will see the use of this feature flag: `target-worker-filters-v2-hcp`. That will control the differences between `ent` and `hcp`. This branch is only for the `ent` version of Boundary.

Reply:
Updated this logic. Moved to `yielding` SingleFilter or DualFilter.